src/go/k8s/cmd/main.go

```diff
@@ -183,7 +183,8 @@ func main() {
 	flag.StringSliceVar(&additionalControllers, "additional-controllers", []string{""}, fmt.Sprintf("which controllers to run, available: all, %s", strings.Join(availableControllers, ", ")))
 	flag.BoolVar(&operatorMode, "operator-mode", true, "enables to run as an operator, setting this to false will disable cluster (deprecated), redpanda resources reconciliation.")
 	flag.BoolVar(&enableHelmControllers, "enable-helm-controllers", true, "if a namespace is defined and operator mode is true, this enables the use of helm controllers to manage fluxcd helm resources.")
-
+	flag.IntVar(&helmCacheMaxSize, "helm-cache-max-size", 0,
```

Comment:
Can we default this to a non-zero value given that zero is what was causing issues in the first place?

Let's also document that `0` means an unlimitted cache size as that's not noted anywhere.

Reply:
0 means no cache, not unlimitted.

Reply:
I am using the same default used by flux, this is why.

Reply:
Thaaaaaaaats very confusing. Please document that in some way.